handlers/network_handler.py

```diff
@@ -0,0 +1,20 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
+
+
+def handle(event, context):
+    response = None
+    if event['pathParameters'] is not None:
+        response = redis_client.get(event['pathParameters']['hash']).decode()
+    else:
+        response = json.dumps([k.decode() for k in redis_client.keys()])
+
+    return {
+        "statusCode": 200,
+        "body": response,
```

Comment:
It seems the response type is different when `pathParameters` is sent or no. I feel we should always return the same format, no matter the parameters. If we need two different return types, maybe we should create to endpoints.

Reply:
Did it!